ceph/rados/core_workflows.py

```diff
@@ -1264,3 +1264,73 @@ def get_osd_stat(self):
         osd_stats = self.run_ceph_command(cmd=cmd)
         log.debug(f" The OSD Statistics are : {osd_stats}")
         return osd_stats
+
+    def run_pool_sanity_check(self):
+        """
+        Runs sanity on the pools after triggering scrub and deep-scrub on pools, waiting 600 Secs
+
+        This method is used to assess the health of Pools after any operation, where in a scrub and deep scrub is
+        triggered, and the method scans the cluster for few health warnings, if generated
+
+        Returns: True/Pass for Pass/Fail
+        """
+        self.run_scrub()
+        self.run_deep_scrub()
+        time.sleep(10)
```

Comment:
Instead of sleep check that scrub and deep-scrub is actually happening in the cluster.

Reply:
Completion of scrub and checking the results is not part of test workflow. I believe that is being covered as part of other tests. 

If the check is a necessity, would incorporate the changes. Let me know.